mapper/orm/src/main/java/org/hibernate/search/mapper/orm/search/loading/impl/MutableEntityLoadingOptions.java

```diff
@@ -6,11 +6,17 @@
  */
 package org.hibernate.search.mapper.orm.search.loading.impl;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.persister.entity.EntityPersister;
 import org.hibernate.search.util.common.impl.Contracts;
 
 public class MutableEntityLoadingOptions {
 	private int fetchSize;
 
+	private List<EntityGraphHint> entityGraphHints;
```

Comment:
This is just my curiosity, is it possible to define multiple graphs on the same query instance?

Reply:
No, but theoretically you could target multiple, unrelated entity types with the same query, in which case you could define multiple graph, one for each entity type.